Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.py

```diff
@@ -5254,6 +5292,35 @@ def list_papi_property_by_hostname_command(client: Client,
     return human_readable, context_entry, raw_response
 
 
+# Created by D.S. 2023-03-30
+@logger
+def list_siteshield_maps_command(client: Client) -> Tuple[str, Dict[str, Any], Union[List, Dict]]:
+    """
+        Returns a list of all Site Shield maps that belong to your account.
+
+    Args:
+        client:
+
+    Returns:
+        human readable (markdown format), entry context and raw response
+    """
+
+    raw_response: Dict = client.list_siteshield_maps()
```

Comment:
Hi Merit, the logic is written in this way to keep a more standard coding practice. Please keep it as is if you don't mind.

Reply:
yes, in our standard coding practice, that's what we do, and since Akamai WAF integration is xsoar supported, it needs to fulfill the requirement i mentioned, so please make the changes accordingly.
thanks!